src/stream/Middleware.ts

```diff
@@ -179,7 +179,7 @@ export class OrderedHistoryResumer extends MiddlewareBase {
     // further back in the stream outside of the given page. A larger page size reduces this likelihood but
     // doesn't solve it. The solution would require paging back 2 mins further to check for any such messages.
     // This is sufficiently low likelihood that this can be ignored for now.
-    this.historicalMessages.sort((a, b) => this.reverseOrderer(a.id, b.id));
+    this.historicalMessages.sort((a, b) => this.reverseOrderer(a.id || '', b.id || ''));
```

Comment:
Similar question around handling these values if empty?

Empty string does won't break the lexicographicOrderer, although both will be treated as strings not numbers

Reply:
Why are these optional now?

Reply:
Because the `Message` type comes from the ably package and the id is now optional